protocol/governance/contracts/modules/core/ElectionModuleSatellite.sol

```diff
@@ -0,0 +1,78 @@
+//SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {SetUtil} from "@synthetixio/core-contracts/contracts/utils/SetUtil.sol";
+import {CrossChain} from "@synthetixio/core-modules/contracts/storage/CrossChain.sol";
+import {OwnableStorage} from "@synthetixio/core-contracts/contracts/ownership/OwnableStorage.sol";
+import {IElectionModule} from "../../interfaces/IElectionModule.sol";
+import {IElectionModuleSatellite} from "../../interfaces/IElectionModuleSatellite.sol";
+import {ElectionCredentials} from "../../submodules/election/ElectionCredentials.sol";
+import {CouncilMembers} from "../../storage/CouncilMembers.sol";
+
+contract ElectionModuleSatellite is IElectionModuleSatellite, ElectionCredentials {
+    using SetUtil for SetUtil.AddressSet;
+    using CrossChain for CrossChain.Data;
+    using CouncilMembers for CouncilMembers.Data;
+
+    uint256 private constant _CROSSCHAIN_GAS_LIMIT = 100000;
+
+    function initCouncilMembers(
```

Comment:
im guessing this function has to be called with the same parameters on all chains (including mothership?)